### PR TITLE
add run/insert first code block actions when there are multiple

### PR DESCRIPTION
**Pull request by meganrogge**
fixes #207604

Screenshot 2024-03-28 at 3 10 24 PM 
Screenshot 2024-03-28 at 3 17 17 PM

### CODE REVIEW

File: src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChat.ts [hunk 1]
```diff
@@ -64,6 +67,9 @@ export namespace TerminalChatContextKeys {
 	/** The type of chat response, if any */
 	export const responseContainsCodeBlock = new RawContextKey<boolean>(TerminalChatContextKeyStrings.ChatResponseContainsCodeBlock, false, localize('chatResponseContainsCodeBlockContextKey', "Whether the chat response contains a code block."));
 
+	/** The type of chat response, if any */
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChat.ts:70]:
```suggestion
	/** The chat response contains multiple code blocks */
```

File: src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChat.ts [hunk 2]
```diff
@@ -64,6 +67,9 @@ export namespace TerminalChatContextKeys {
 	/** The type of chat response, if any */
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChat.ts:67]:
```suggestion
	/** The chat response contains a single code block */
```

File: src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChat.ts [hunk 3]
```diff
@@ -61,9 +64,12 @@ export namespace TerminalChatContextKeys {
 	/** Whether the terminal chat agent has been registered */
 	export const agentRegistered = new RawContextKey<boolean>(TerminalChatContextKeyStrings.ChatAgentRegistered, false, localize('chatAgentRegisteredContextKey', "Whether the terminal chat agent has been registered."));
 
-	/** The type of chat response, if any */
+	/** The chat response contains a single code block */
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChat.ts:67]:
```suggestion
	/** The chat response contains at least one code block */
```

File: src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatController.ts [hunk 4]
```diff
@@ -327,7 +329,10 @@ export class TerminalChatController extends Disposable implements ITerminalContr
 				this._chatAccessibilityService.acceptResponse(responseContent, accessibilityRequestId);
 				const containsCode = responseContent.includes('```');
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatController.ts:330]: note we still need `containsCode` check above bc have to call `updateChatMessage` for the code blocks to get rendered

**Review by Tyriar — APPROVED** [review]

**Pull request merged**